CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
-cmake_minimum_required (VERSION 3.14)
+cmake_minimum_required (VERSION 3.5)
 project(tim-vx LANGUAGES C CXX)
 
-option(BUILD_SHARED_LIBS                "Build using shared libraries"          ON)
+#option(BUILD_SHARED_LIBS                "Build using shared libraries"          OFF)
+set(BUILD_SHARED_LIBS OFF)
```

Comment:
`option(BUILD_SHARED_LIBS OFF` triggers a linking problem (https://github.com/fengyuentau/opencv/runs/5713925837?check_suite_focus=true), so we use `set(BUILD_SHARED_LIBS OFF)` instead.